zubhub_frontend/zubhub/src/assets/js/styles/views/page_wrapper/pageWrapperStyles.js

```diff
@@ -10,7 +10,7 @@ const styles = theme => ({
     fontSize: '1.15rem',
   },
   mainContainerStyle: {
-    maxWidth: '2000px',
+    maxWidth: '6000px',
```

Comment:
I don't think its a good idea there should be some other issue we need to figure it out

Reply:
can you try `100vw` its 100% width of the device